src/resources/UsageAnalytics/Read/Snowflake/SnowflakeInterfaces.ts

```diff
@@ -28,3 +28,7 @@ export interface GetCreditUsageParams {
 export interface SnowflakeCreditUsageModel {
     creditsUsed: number;
 }
+
+export interface SnowflakeReaderAccountModel {
+    snowflakeReaderAccountStatus: string;
```

Comment:
Do we have an enum for the status?

Reply:
I think we could... I'll drop that subtly hehehe:

```
export enum SnowflakeReaderAccountStatus {
    creating = 'CREATING',
    available = 'AVAILABLE',
}
```

Reply:
@jmprovencher Could you validate this?

Reply:
`public enum SnowflakeReaderAccountStatus {
  CREATING,
  AVAILABLE
}
`

Reply:
looks good

Reply:
Add missing JSDoc for this field and you have my approval 👍